test/model/seq2seq/test_quantile_loss.py

```diff
@@ -36,3 +36,18 @@ def test_compute_quantile_loss() -> None:
             - correct_qt_loss[idx]
             < 1e-5
         ), f"computing quantile loss at quantile {q} fails!"
+
+
+def test_compute_weighted_quantile_loss() -> None:
```

Comment:
You should be actually able to combine this with the `test_compute_quantile_loss` function for minimal code change by importing `pytest`. 
You can do the following:
`@pytest.mark.parametrize("quantile_weights", [None, [0.5, 0.5]])
def test_compute_quantile_loss(quantile_weights) -> None:`

and it will iterate the test over both values of `quantile_weights`.